include/zephyr/usb/usbd_msg.h

```diff
@@ -57,6 +61,8 @@ static const char *const usbd_msg_type_list[] = {
 	"Bus reset",
 	"Controller error",
 	"Stack error",
+	"New CDC ACM line coding",
+	"New CDC ACM control line state",
```

Comment:
Nit: this is not necessarily "New" as the line coding or control line state might be exactly the same as before (the notifications are for every SET_LINE_CODING and SET_CONTROL_LINE_STATE).

Reply:
Yes, I am bad at producing human-readable text.